src/components/text-editor/examples/text-editor-custom-triggers.tsx

```diff
@@ -43,13 +46,16 @@ export class TextEditorCustomTriggersExample {
     private triggerState: string = '';
 
     @State()
-    private inputText: string = '';
+    private tagValue: string = '';
 
     @State()
     private isPickerOpen: boolean = false;
 
     @State()
-    private insertMode: 'text' | 'chip' = 'text';
+    private insertMode: 'text' | 'chip' | 'html' = 'text';
+
+    @State()
+    private outputMode: 'markdown' | 'html' = 'markdown';
```

Comment:
I updated this example so it's possible to change output mode. (The editor needs to be destroyed and recreated for that to work of course, and we reset the value to an empty string.)

The fixup for this change, in case anyone wants to have a look after I have already squashed it, is: c8fe03aa4981b7fc90c293652c3b58da94753787